sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_common/_eval_utils

```diff
@@ -0,0 +1,66 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+# Contains a collection of functions that are used by evaluators to specify internal
+# logic, espcecially w.r.t to conversation -> input conversion.
+
+from typing import Dict, List, Func
+
+
+# TODO refactor to use inspection instead of bools to determine what inputs to use?
+def get_per_turn_converter(include_query: bool = True, include_response: bool = True, include_context: bool = True, eval_last_turn_only = False) -> Func:
```

Comment:
this file doesn't have .py ext

Reply:
Removed the file, its functionality was rolled into eval_base and was never meant to be committed,